### PR TITLE
Alerting: Explore button in Insights view

### PR DESCRIPTION
**Pull request by ppcano**
Changes:

- Added an `Explore` button to the Insights panel.
- Fixed queries in four Insights panels.
- Added a title tooltip to describe Alerting Insights.

The [Insights documentation](https://grafana.com/docs/grafana-cloud/alerting-and-irm/alerting/manage-notifications/view-insights/) was recently updated.  For additional documentation updates, I’d prefer to handle them in a separate PR.

**Preview**

- Your instance can be accessed at: https://ephemeral1511182196496ppcano.grafana-dev.net
- [How to access / How to update instance config / How to build a specific branch](https://github.com/grafana/hosted-grafana/wiki/Ephemeral-instances#choosing-which-oss-and-enterprise-branches-are-used-to-build-the-instance)

https://github.com/user-attachments/assets/fc963de0-ed78-4fda-92dc-311c7bc49dce

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.
- [ ] Add to changelog

### CODE REVIEW

File: public/app/features/alerting/unified/home/Insights.tsx [hunk 1]
```diff
@@ -173,7 +176,29 @@ export function getInsightsScenes() {
     controls: [
       new SceneReactObject({
         component: SectionSubheader,
-        props: { children: <div>Monitor the status of your system.</div> },
+        props: {
+          children: (
+            <React.Fragment>
+              <Text>
+                {' '}
```
Comment (soniaAguilarPeiron) [inline public/app/features/alerting/unified/home/Insights.tsx:183]: why do we need these spaces?

Comment (ppcano) [reply public/app/features/alerting/unified/home/Insights.tsx]: No. Removed by https://github.com/grafana/grafana/pull/96496/commits/5234df526a18dac088d3a3c9ea5ab9fac3f85e8a

**Review by soniaAguilarPeiron — APPROVED** [review]
LGTM! Great work!🚀

**Pull request merged**